src/main/kotlin/apply/ui/admin/SelectionView.kt

```diff
@@ -37,13 +38,17 @@ class SelectionView(
 
     private fun createMenu(): Component {
         return HorizontalLayout(
-            createSearchBar {
-                removeAll()
-                add(createTitle(), createMenu(), createGrid(applicantService.findByValue(it)))
-            },
-            createSuccessButton("다운로드") {
-                // Todo: 엑셀 다운로드
-            }
+                createSearchBar {
+                    removeAll()
+                    add(createTitle(), createMenu(), createGrid(applicantService.findByValue(it)))
+                },
+                createSuccessButton("다운로드") {
+                    val excel = { downloadService.createExcelBy(recruitmentId) }
+                    val registration = VaadinSession.getCurrent()
+                            .resourceRegistry
+                            .registerResource(StreamResource("$recruitmentId.xlsx", excel))
```

Comment:
```suggestion
                            .registerResource(StreamResource("${recruitmentService.getById(recruitmentId).title}.xlsx", excel))
```

이 부분을 이렇게 수정하면 사용성이 올라갈 거 같아요..!
지금은 recruitmentId를 파일 제목으로 사용하기에 1.xlsx 이렇게 나오지만 수정시 웹 백엔드 2기.xlsx로 나오게 할 수 있으니까요!

위에도 title구하는 로직이 반복되는데 중복 제거는 시카의 취향대로~!

Reply:
좋은 방법이네요 👍 덕분에 vaadin을 사용하는 이유에 대해서 다시 한 번 생각해보았네요
현재는 recruitmentId만 사용해서 웹백앤드2기.xlsx만 사용하지만, 추후에 웹백앤드2기-프리코스 와 같이 확장이 필요한데요
이 부분도 논의해볼 필요가 있을 것 같네요